pastel/art_ticket.go

```diff
@@ -1,7 +1,31 @@
 package pastel
 
-// ArtTicket represents pastel art ticket.
+// Ticket is Pastel NFT ticket structure
+type Ticket struct {
```

Comment:
You cannot occupy name `Ticket` because Pastel has different tickets. for example `IDTicket`.
All structures related to Art must be prefixed with `Art`.

```suggestion
type ArtTicket struct {
     ArtTicketData ArtTicketData `json:"-"`
}

type ArtTicketData struct {
     AppTicketData ArtAppTicket `json:"-"`
}
```

Reply:
![image](https://user-images.githubusercontent.com/46045452/121404931-e2e5ce00-c975-11eb-9627-4ce03ba0d002.png)
This is the outer ticket structure which has a field art_ticket. If I name the outer ticket as art_ticket then what will we do with the inner one? Should we name the outer as PastelTicket or rename the inner as ArtRegTicket?

Reply:
I advice you to work with real data. from `pastel-cli`, because in fact it can turn out a little differently. Or how are you going to test it? 
So, please show the result of Pastel API, and then we can continue to discuss.

Reply:
So far you can use `json` tag according to documented structure, but the Golang structure should be called correctly 

example:
```go
type ArtTicket struct {
     ......
     ArtTicketData ArtTicketData `json:"art_ticket"`
}
```